connect/src/com/telenor/connect/ConnectSdk.java

```diff
@@ -518,17 +518,22 @@ public static synchronized void sdkReinitializeMobileConnect() {
      * Initialize components common to both Mobile Connect and ConnectID SDK profiles
      */
     private static synchronized void initializeCommonComponents() {
+        if (Build.VERSION.SDK_INT < Build.VERSION_CODES.LOLLIPOP) {
```

Comment:
this has to be under getting connectivityManager
otherwise on older devices connectivityManager will be null and isCellularDataNetworkConnected is useless